src/utils/leaseEvidenceSortAndFilter.ts

```diff
@@ -0,0 +1,49 @@
+// TODO - tsconfig replace CommonJS with esnext
```

Comment:
Will it matter having the leases filter in this branch?

Reply:
I'll delete this. In `tsconfig.json` under `"compilerOptions"` I changed `"module": "esnext"` to `"module": "CommonJS"` so as to test the filter function I was using. [This](https://stackoverflow.com/questions/56546541/typescript-syntaxerror-unexpected-token) SO post helped me a bit.